cmd/eth/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"sort"
+)
+
+type cmd struct {
+	desc string              // command description
+	do   func(args []string) // do it
+	fs   flag.FlagSet
+}
+
+func fatalf(f string, args ...interface{}) {
+	fmt.Fprintf(os.Stderr, f, args...)
+	os.Exit(1)
+}
+
+var subcommands = map[string]*cmd{
+	"balance": cmdbal,
+}
+
+func usage() {
```

Comment:
I don't really like printing usage to stderr because it makes you do `eth -help 2>&1 | less` instead of `eth -help | less`.

Reply:
done